Dockerfile

```diff
@@ -76,4 +76,4 @@ RUN apt-get purge -y
 
 USER plasma
```

Comment:
I think we should not use `plasma` through out this code. "plasma" is a design framework.

Terms which should be used is:
 - child chain - for the name of the blocks/ledger/data
 - child chain server - for the application "running" the child chain 
 - operator - the entity that's running the child chain server - which is what probably fits in this particular spot.

There are a couple of occurrences of "plasma" throughout this PR

Reply:
That wasn't introduced via this PR.

That being said, I'm happy to modify it in separate change if you feel it should be changed. I think you've slightly misunderstood though, this is for the user of the running container so that we don't run the application as root. I don't want to follow that naming convention for creating Linux system users. It would result in three separate Dockerfiles for running the same code, which is very inefficient.

I propose elixir-user, or similar.

Reply:
`elixir-user` is good of course, I'm just after cleaning up the "plasma-terminology" here, so anything that doesn't touch this is good.

I'd clean up now however, since more `plasma` occurrences are introduced in this PR.

And sorry about late noticing, I know this's been around since before this PR.